### ISSUE TITLE
COVID-19 home page alerts and messages

### ISSUE THREAD

**Issue by Ig-Rebollo** [kobotoolbox/kobotoolbox.org#35]
Create mockups to display KoBo's response to COVID-19 in the homepage.

Messaging may reference lifting data storage limits for projects related to the pandemic.

See below initial mockups with proposal:
![COVID19-homepage-01](https://user-images.githubusercontent.com/22472326/78840867-ebd1bd00-79c9-11ea-9393-6df287413622.png)
![COVID19-homepage-02](https://user-images.githubusercontent.com/22472326/78840876-f1c79e00-79c9-11ea-938e-50eef6f72952.png)

**Comment by tinok** [comment]
Thanks @Ig-Rebollo. I think that looks great. Here is some copy for the two boxes: 

```
Banner: 

(Keep sentence 1 as is). If you have other requests or suggestions how we can help, [please let us know](#scrolldowntobox).



Box:

[Repeat sentence 1 from box.] This only applies to users of https://kf.kobotoolbox.org; humanitarian users of https://kobo.humanitarianresponse.info already enjoy completely unlimited storage. If applicable, [please fill in this short form](https://ee.kobotoolbox.org/single/::VNLm4eav) to request more storage for your project.

Do you need support/features/fixes to use KoBoToolbox in COVID-19 projects? Does your COVID-19 project need help related to data collection? Please let us know, we may be able to find funding or connect you with volunteers. [Please take this short survey to let us know](https://ee.kobotoolbox.org/single/::VNLm4eav).

```

**Comment by Ig-Rebollo** [comment]
@tinok here the updated mockups with the new copy. A few things to note:

I made two options for the more elaborated section. One is with your exact copy, using links; the other one makes some very small changes to the text in order to place the links as buttons. This is because with the colour of the background, the normal link colour does not offer enough contrast to pass standard accessibility requirements. We would therefore have to use a darker blue only for these links. By using buttons (and making the type of account italic instead of a link) we avoid that problem. In any case, both alternatives would work fine, though my preference would be the buttons. 

In terms of specs, as soon as the mockups get your green light, I can upload the icon file in svg, the colour codes I used (most of them already on the site), and anything else that might be needed.

Let me know what you think.

![COVID19-homepage-01](https://user-images.githubusercontent.com/22472326/79472533-f4ca1c00-7fd1-11ea-8913-3f1abcfb592d.png)
![COVID19-homepage-02](https://user-images.githubusercontent.com/22472326/79472552-f8f63980-7fd1-11ea-8753-fdb2ca13e120.png)
![COVID19-homepage-03](https://user-images.githubusercontent.com/22472326/79472558-fa276680-7fd1-11ea-97c5-366e5565a49d.png)
